api/config/config.json

```diff
@@ -0,0 +1 @@
+{"googleApiKeys":{"CLIENT_ID": null,"CLIENT_SECRET": null,"REDIRECT_URIS":["https://developers.google.com/oauthplayground"],"USER":"sce.sjsu@gmail.com","REFRESH_TOKEN":null}}
```

Comment:
can you make this `config.example.json` and have it replace the existing `config.example.js`?

Reply:
These changes are now in PR #528. Added a `config.json` to this PR to get Travis to build but it seems to still fail